.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+
+A clear and concise description of what the problem is. Ex. When trying to do [...] I need to [...] because [...] is missing"
```

Comment:
A clear and concise description of what the problem is. Ex. "When trying to [...] I need to [...] because [...] is missing"